src/xbot/common/subsystems/BaseDriveSubsystem.java

```diff
@@ -5,10 +5,13 @@
 
 import xbot.common.command.BaseSubsystem;
 import xbot.common.controls.actuators.XCANTalon;
+import xbot.common.logging.LoggingLatch;
 import xbot.common.math.XYPair;
 
 public abstract class BaseDriveSubsystem extends BaseSubsystem {
-
+    
+    private final LoggingLatch baseDriveSubsystemLoggingLatch = new LoggingLatch(this.getName(), "XCanTalon(s): null");
```

Comment:
Can we make this message more descriptive? e.g. "CAN Talons null in drive subsystem"